hieradata/common.yaml

```diff
@@ -635,6 +635,26 @@ govuk_ci::master::pipeline_jobs:
   ubuntu_unused_kernels:
     repo_owner: 'gds-operations'
 
+govuk_ci::master::ci_agents:
+  ci-agent-1:
+    agent_hostname: 'ci-agent-1.ci'
+    description: 'Build agent with docker'
+    labels: 'mongodb-2.4 docker'
```

Comment:
Would you please do me a solid and amend the labels so they match the labels in our Jenkinsfiles?

i.e. `mongo-3.2` becomes `mongodb-3.2`